pkg/reconciler/common/images.go

```diff
@@ -45,13 +46,12 @@ var (
 func ImageTransform(registry *v1alpha1.Registry, log *zap.SugaredLogger) mf.Transformer {
 	return func(u *unstructured.Unstructured) error {
 		switch u.GetKind() {
-		// TODO need to use PodSpecable duck type in order to remove duplicates of deployment, daemonSet
 		case "Deployment":
```

Comment:
Not sure if I should check if the type implements `WithPod` or if we should check the `kind` manually.
Left that part as is for now.